.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
Seems reasonable. As you can tell from my other gitignore commit, I'm an IntelliJ user so good to have some feedback from real VS Code users :)